lua/harpoon/ui.lua

```diff
@@ -163,8 +163,14 @@ function M.nav_file(id)
         return
     end
 
+    local filename = nil
     local mark = Marked.get_marked_file(idx)
-    local buf_id = vim.fn.bufnr(vim.fn.getcwd() .. "/" .. mark.filename, true)
+    if mark.filename:sub(1, 1) == "/" then
+        filename = mark.filename
+    else
+        filename = vim.fn.getcwd() .. "/" .. mark.filename
```

Comment:
I think you could use `vim.loop.cwd()` which backed by libuv

Reply:
That same code is in the pre-image so I didn't change it, but if we think that's useful I can add an additional commit to swap over.

Yay / nay?

Reply:
yep, I know this is none of your "fault" so to say haha
simply noticed that this was still the old VimScript function, while newer APIs (and in this case especially because async) are preferred :)

Reply:
Sweet, thanks for review ;-) I appreciate the free nvim education. I pushed in a new commit that swaps over to `vim.loop.cwd()`.